docs/deploying/docker.html

```diff
@@ -103,22 +103,26 @@ <h4>Currently in Beta - <a href="../../old-learn.html">View old docs</a></h4>
     <div id="doc-container" class="docs-item-4 docs-window">
       <main>
         <h1 class="heading-1">Deploy to the Docker platform</h1>
-        <p> Docker provides tooling and a platform to manage the lifecycle of lightweight virtualization packages known as containers.
+        <p> Docker provides tooling and a platform to manage the lifecycle of lightweight virtualization packages, known as containers.
         In this guide, we will cover the steps necessary to Dockerize a Kitura application. This includes how to build the Docker image and how to create
         the container instance from that image.</p>
 
-        <p>A full description of the Docker platform can be found on the Docker <a href="https://docs.docker.com/get-started/" >website </a>, including the installation instructions for the
-        supported operating systems. This guide will be based on the two Dockerfiles generated by the <a href="https://www.kitura.io/docs/getting-started/create-server-cli.html">Kitura init </a>command. </p>
+        <p>A full description of the Docker platform can be found on the Docker <a href="https://docs.docker.com/get-started/" >website</a>, including installation instructions.
+           This guide is based on the two Dockerfiles generated by the Kitura CLI's <a href="https://www.kitura.io/docs/getting-started/create-server-cli.html">kitura init</a> command. </p>
 
         <h2 class="heading-2"><span class="blue-text">Step 1:</span> Build compile image</h2>
         <p> As the Swift compiler is large and not required for running compiled Kitura applications,
-            we separate the Docker images for application compilation and runtime into a build image and a run image.
-            The <strong>Kitura init</strong> command supports this two step build process by generating two Dockerfiles for your Kitura project.
-            <strong>Dockerfile-tools</strong> supports the compilation step and <strong>Dockerfile</strong> supports the building of the Docker image for running your Kitura application.</p>
+            the Docker images for application compilation and runtime are separated into a build image and a run image.
+            The <strong>kitura init</strong> command supports this two step build process by generating two Dockerfiles for your Kitura project:
+        <ul>
+          <li><strong>Dockerfile-tools</strong> supports the compilation step</li>
```

Comment:
I think we can expand this out a bit. Something like...

Dockerfile-tools: building this Dockerfile produces a Docker image containing everything needed to compile your Kitura application. Running the image will compile your Kitura application.

Dockerfile: building this Dockerfile produces a Docker image containing your compiled application. Running the image will run your application.

Reply:
I've pushed the requested changes @ianpartridge please can you re-review?